detox/local-cli/detox-test.js

```diff
@@ -24,11 +24,13 @@ program
   .option('-p, --platform [ios/android]',
     'Run platform specific tests. Runs tests with invert grep on \':platform:\', '
     + 'e.g test with substring \':ios:\' in its name will not run when passing \'--platform android\'')
+  .option('-f, --file [path]',
+    'Specify test file to run')
   .parse(process.argv);
 
 const config = require(path.join(process.cwd(), 'package.json')).detox;
 
-const testFolder = getConfigFor('specs', 'e2e');
+const testFolder = getConfigFor('file', 'specs', 'e2e');
```

Comment:
`getConfigFor` has two params, this puts `file` as key, pushing `specs` as default value, ignoring `e2e` altogether. Am I missing something ?

Reply:
I can be a good idea to extend `getConfigFor` to support multiple keys